functional/cucumber.json

```diff
@@ -0,0 +1,97 @@
+[
```

Comment:
what is this file? probably pushed to repo by mistake?

Reply:
We added this functional/cucumber.json.  This is just a place holder file to pass the build task.   Because,  at the time of building the project (clean build) some how it is looking for cucumber.json file. If we didn’t provide this file, build is failing.
During functional task, the actual file with data overtires this placeholder.
Do you think is there any harm adding a placeholder file?
if yes, please suggest. Thanks

Reply:
Could we not try an empty place holder file, then?

Reply:
Ben suggested a simple and great solution. generateCucumberReports.enabled = false in test task and enable the same  in functional task. It work's in local. Checking in pipeline.

Reply:
That's cool. Thank you.